File: src/main/java/project/rpg/listeners/EntityTakeDamageEventListener.java
package project.rpg.listeners;

import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.entity.EntityDamageEvent;
import project.rpg.Rpg;


public class EntityTakeDamageEventListener implements Listener {

    private final Rpg _plugin;

    public EntityTakeDamageEventListener(Rpg plugin){
        _plugin = plugin;
    }

    @EventHandler
    public void onEntityTakeDamage(EntityDamageEvent event) {
        if (event.getEntity() instanceof Player) {
            event.setDamage(event.getDamage()/100.0);
            _plugin.actionBar.updateActionBar((Player) event.getEntity());
        }
    }
}

File: src/main/java/project/rpg/events/listeners/EntityTakeDamageEventListener.kt
package project.rpg.events.listeners

import org.bukkit.entity.Player
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.entity.EntityDamageEvent
import project.rpg.Rpg

class EntityTakeDamageEventListener(private val _plugin: Rpg) : Listener {
    @EventHandler
    fun onEntityTakeDamage(event: EntityDamageEvent) {
        if (event.entity is Player) {
            event.damage = event.damage / 100.0
            _plugin.actionBar.updateActionBar(event.entity as Player)
            _plugin
        }
    }
}